multiqc/search_patterns.yaml

```
@@ -24,6 +24,9 @@ bamdst/coverage:
 bamtools/stats:
   contents: "Stats for BAM file(s):"
   num_lines: 10
+bamutil_clipoverlap:
```

Should be `tool/subcommand`. Needs to tie into module code too.

```suggestion
bamutil/clipoverlap:
```